Assets/__Scripts/MapEditor/Audio/MetronomeHandler.cs

```diff
@@ -73,7 +92,7 @@ private void RunAnimation(float inferredBpm)
 
     private void OnPlayToggle(bool playing)
     {
-        if (metronomeVolume == 0) return;
+        if (Settings.Instance.MetronomeVolume == 0) return;
```

Comment:
Can probably replace this while we're here
```suggestion
        if (!metronomeActive) return;
```